src/fstab-generator/fstab-generator.c

```diff
@@ -838,6 +874,11 @@ static int add_sysroot_mount(void) {
                 log_debug("Skipping root directory handling, as gpt-auto was requested.");
                 return 0;
         }
+        else if (streq(arg_root_what, "fstab")) {
```

Comment:
systemd's coding style requires the `else` to be on the same line as `}`, i.e.

```suggestion
        } else if (streq(arg_root_what, "fstab")) {
```

See also: https://systemd.io/CODING_STYLE/

Reply:
Oops I even read that document, but I guess habit still took over. Thank you for the catch!